lib/msf/ui/console/command_dispatcher/exploit.rb

```diff
@@ -100,62 +91,22 @@ def cmd_run_tabs(str, words)
   # Launches exploitation attempts.
   #
   def cmd_exploit(*args)
-    force = false
-    module_opts = []
+    return false unless (args = parse_exploit_opts(args))
```

Comment:
This is kind of "bad" Ruby IMO - `unless <condition>` with an internal assignment is a bit more legible as `args = ..` and `if args` subsequently.

Reply:
Agreed, decided to align with the existing pattern for consistency - but might clean it up in a separate PR :+1: